reporting.md

```diff
@@ -0,0 +1,615 @@
+%%%
+
+   Title = "SMTP TLS Reporting"
+   abbrev = "SMTP-TLSRPT"
+   category = "std"
+   docName = "draft-ietf-uta-smtp-tlsrpt-00"
+   ipr = "trust200902"
+   area = "Applications"
+   workgroup = "Using TLS in Applications"
+   keyword = [""]
+
+   date = 2016-04-18T00:00:00Z
+
+   [[author]]
+   initials="DM"
```

Comment:
Confusing as it is, the "initials" field is supposed to be the first name initial, e.g. 

   [[author]]
   initials="M."
   surname="Risher"
   fullname="Mark Risher"
   organization="Google, Inc"
     [author.address]
     email="risher (at) google (dot com)"